subsys/fw_metadata/Kconfig

```diff
@@ -18,11 +18,13 @@ config FW_VALIDATION_METADATA_OFFSET
 	  aligned to closest word.
 
 config FW_FIRMWARE_INFO_OFFSET
-	hex "The location of firmware info inside firmware."
-	default 0x800
+	hex "The location of firmware info inside this firmware."
```

Comment:
Would it ever be unclear what this firmware is related to incase multiple systems use it?

Reply:
Changed to "inside the current firmware image."